ecommerce/OrderController.php

```diff
@@ -383,7 +383,7 @@ public function add_comment() {
 		$params = array(
 			'order_id'   => Input::post( 'order_id' ),
 			'meta_key'   => OrderActivitiesModel::META_KEY_COMMENT,
-			'meta_value' => Input::post( 'comment' ),
+			'meta_value' => Input::post( 'meta_value' ),
```

Comment:
I believe the API request payload contains the input key named `comment` instead of `meta_value`. You are responsible to convert this into your required structue.